apps/frontend/package.json

```diff
@@ -19,6 +19,7 @@
     "chess.js": "^1.0.0-beta.8",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
+    "framer-motion": "^11.1.7",
```

Comment:
very heavy library to add for such a small thing
Can u use a simple gif instead?

Reply:
Sure I will try to look for a better solution for this.